kotlin-react-dom/src/main/kotlin/react/dom/experimental/render.kt

```diff
@@ -0,0 +1,16 @@
+package react.dom.experimental
+
+import org.w3c.dom.Element
+import react.dom.render
+import react.experimental.Render
+import react.experimental.createElement
+
+fun render(
+    container: Element,
```

Comment:
`Element?` is better here, because e.g. `document.getElementById` returns `Element?`